messaging/service-worker.js

```diff
@@ -10,21 +10,23 @@ function initInSw() {
   // Give the service worker access to Firebase Messaging.
   // Note that you can only use Firebase Messaging here. Other Firebase libraries
   // are not available in the service worker.
-  importScripts('https://www.gstatic.com/firebasejs/9.2.0/firebase-app.js');
-  importScripts('https://www.gstatic.com/firebasejs/9.2.0/firebase-messaging.js');
+  importScripts("https://www.gstatic.com/firebasejs/8.10.1/firebase-app.js");
+  importScripts(
+    "https://www.gstatic.com/firebasejs/8.10.1/firebase-messaging.js"
+  );
 
   // Initialize the Firebase app in the service worker by passing in
   // your app's Firebase config object.
   // https://firebase.google.com/docs/web/setup#config-object
   firebase.initializeApp({
-    apiKey: 'api-key',
-    authDomain: 'project-id.firebaseapp.com',
-    databaseURL: 'https://project-id.firebaseio.com',
-    projectId: 'project-id',
-    storageBucket: 'project-id.appspot.com',
-    messagingSenderId: 'sender-id',
-    appId: 'app-id',
-    measurementId: 'G-measurement-id',
+    apiKey: "api-key",
+    authDomain: "project-id.firebaseapp.com",
+    databaseURL: "https://project-id.firebaseio.com",
+    projectId: "project-id",
+    storageBucket: "project-id.appspot.com",
+    messagingSenderId: "sender-id",
+    appId: "app-id",
+    measurementId: "G-measurement-id",
```

Comment:
Are the double quotes necessary? Most (if not all) of our js snippets use single quotes, and I'd prefer to keep them consistent. (applies throughout)

Reply:
Probably my default prettier settings. If it makes sense to do it for the repo, it might be good to add a [prettierrc file](https://github.com/firebase/firebase-js-sdk/blob/master/.prettierrc#L2)

Reply:
Updated the PR